### PR TITLE
update comments.html template

### PR DESCRIPTION
**Pull request by g1039**

### CODE REVIEW

**Comment by g1039** [comment]
@miltontony @hodgestar @rudigiesler @saeedmb8 Ready for review

File: tuneme/templates/comments/comments.html [hunk 1]
```diff
@@ -1,5 +1,4 @@
-{% extends "core/_article_page.html" %}
```
Comment (rudigiesler) [inline tuneme/templates/comments/comments.html:1]: Why are we removing this line?

**Comment by rudigiesler** [comment]
We probably also want to update/add the unit tests to ensure that this is displayed properly. Otherwise looks good to me.

**Comment by g1039** [comment]
@rudigiesler @saeedmb8 @hodgestar @miltontony Ready for review

**Comment by rudigiesler** [comment]
:+1: Looks good to me.

**Pull request merged**